src/core/server/core_usage_data/core_usage_stats_client.ts

```diff
@@ -150,7 +150,7 @@ export class CoreUsageStatsClient {
     const { createNewCopies, overwrite } = options;
     const counterFieldNames = [
       `createNewCopiesEnabled.${createNewCopies ? 'yes' : 'no'}`,
-      `overwriteEnabled.${overwrite ? 'yes' : 'no'}`,
+      ...(!createNewCopies ? [`overwriteEnabled.${overwrite ? 'yes' : 'no'}`] : []), // the overwrite option is ignored when createNewCopies is true
```

Comment:
While we'll ignore `overwriteEnabled.yes` and `overwriteEnabled.no` fields in each report sent to the telemetry cluster (legacy `all-xpack-phone-home` index), we'll still have a chance of reporting the field in daily snapshot (legacy `xpack-phone-home` index).

Reply:
Sorry for the confusion, this is just in the function to increment the counters. The intent here is that if the Import API is called with `createNewCopies: true`, then regardless of whether the `overwrite` option is specified, the counter field for `overwriteEnabled.yes`/`overwriteEnabled.no` will not be incremented.

We do want the daily snapshot to collect all the fields! 😄